### PR TITLE
HA DB for each PR.

### PR DESCRIPTION
**Pull request by Sybrand**
- Templates and scripts for adding database for each deployment.

### CODE REVIEW

**Comment by Sybrand** [comment]
Testing cleanup

**Comment by Sybrand** [comment]
Cleanup worked

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -6,6 +6,24 @@ on:
       - main
 
 jobs:
+  prepare-database:
```
Comment (Kyubinhan) [inline .github/workflows/deployment.yml:9]: Any reason why this job gets special treatment?

Comment (Sybrand) [reply .github/workflows/deployment.yml:27]: As we discussed - it runs in parallel with the build. Both build and db now dependant on the secrets.

**Comment by Sybrand** [comment]
Testing cleanup

**Comment by Sybrand** [comment]
Cleanup worked.

**Pull request merged**